### ISSUE TITLE
New project doesn't run on Node 18.17.1

### ISSUE THREAD

**Issue by georgemarrows** [observablehq/framework#1026]
```
18:42 $ node --version
v18.17.1
18:43 $ npm --version
9.6.7
18:43 $ npm run dev

> dev
> observable preview


Unexpected error: Unknown file extension ".ts" for /Users/georgem/work/observable/test1/observablehq.config.ts
```

This works fine with Node 20.11.1. The problem is the origin of my aborted https://github.com/observablehq/framework/pull/1017
I see [the comment](https://github.com/observablehq/framework/pull/1017#issuecomment-1984551130) that you can remove `observablehq.config.ts` but ideally of course you wouldn't have to.
 
Really great looking framework by the way!

<details>
<summary>Project creation steps, if it matters...</summary>

18:49 $ npm init @observablehq
┌   observable create  v1.2.0
│
◇  Where to create your project?
│  test2
│
◇  What to title your project?
│  Test2
│
◇  Include sample files to help you get started?
│  Yes, include sample files
│
◇  Install dependencies?
│  Yes, via npm
│
◇  Initialize git repository?
│  Yes
│
◇  Installed! 🎉

</details>

**Comment by mbostock** [comment]
So to clarify, a TypeScript project configuration file doesn’t work for you on Node 18, but if you switch to JavaScript, it works fine? That’s good to know. And I’ll retitle the issue accordingly if you confirm.

**Comment by mbostock** [comment]
Probably we should just clarify that Node 20.6+ is required for TypeScript config.

**Comment by georgemarrows** [comment]
I don't think I made a choice to get a `observablehq.config.ts`. It was just what I got by default when I ran `npm init @observablehq`. Unless there's something in my Node environment that influenced this somehow?

**Comment by georgemarrows** [comment]
Aha, I see you switched to .js config 👍

**Comment by mbostock** [comment]
Fixed in #1027.
